apps/frontend/components/NewExp.tsx

```diff
@@ -15,6 +15,7 @@ import { PostProcessStep } from './stepComponents/PostProcessStep';
 import { ConfirmationStep } from './stepComponents/ConfirmationStep';
 import { DumbTextArea } from './stepComponents/DumbTextAreaStep';
 
+const DEFAULT_TIMEOUT = 5*60*60; // 5 hours in seconds
```

Comment:
rename to DEFAULT_TRIAL_TIMEOUT_SECONDS (I assume we'll have an experiment-wide timeout later)

Reply:
Changed